app/assets/stylesheets/new_styles/_forms.scss

```diff
@@ -1,91 +1,115 @@
+input,
+input[type=email],
+input[type=text],
+input[type=password],
+textarea { /* Bootstrap reset */
+  &,
+  &:active,
+  &:invalid,
+  &:invalid:required,
+  &:focus,
+  &:active:focus,
+  &:invalid:focus,
+  &:invalid:required:focus {
+    border-color: $border-grey;
+    box-shadow: none;
+    color : $text-dark-grey;
+  }
+}
+
 /* autocomplete colors */
 input:-webkit-autofill{
   background-color: #fff !important;
   background-image: none !important;
 }
 
-
+/* Forms described here are only used on the public pages at the moment */
 form.block-form {
-  display : inline-block;
+  margin: 20px auto;
 
   label {
     color : $text-dark-grey;
+    position: absolute;
```

Comment:
You talk too much @houndci, has someone already told you that ? Oh wait ! No ! You're a bot ! :p